build.sbt

```diff
@@ -301,37 +300,6 @@ lazy val `aws-kernel` = projectMatrix
   )
   .nativePlatform(allNativeScalaVersions, nativeDimSettings)
 
-/**
- * cats-effect specific abstractions of AWS protocol interpreters and constructs
- */
-lazy val aws = projectMatrix
```

Comment:
removed as supporting non-http4s clients in not in the scope of the project anymore.